include/deal.II/matrix_free/operators.h

```diff
@@ -301,7 +301,7 @@ namespace MatrixFreeOperators
    *
    * @author Daniel Arndt, 2016
    */
-  template <int dim, int fe_degree, int n_components = 1, typename Number = double>
```

Comment:
something is wrong on GitHub. This shall not be a part of the PR, as it is already done and that's how the `master` looks like now (with `n_q_points_1d`).